buildSrc/subprojects/parser/src/test/resources/integrationTest/expected.ftl

```diff
@@ -14,10 +14,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+// Extracted from: parserTest/parserImpls.ftl
 <DEFAULT> TOKEN :
 {
 
 }
+
+// Extracted from: parserTest/intermediate/parserImpls.ftl
```

Comment:
Should we have the parsingTest directory structure match the real one? e.g. parserTest/dialects/intermediate/parserImpls.ftl here?

Reply:
Good idea, will make the change.